package.json

```diff
@@ -125,27 +126,27 @@
   "bundlesize": [
     {
       "path": "packages/react-instantsearch/dist/umd/Core.min.js",
-      "maxSize": "7.75 kB"
+      "maxSize": "7.50 kB"
     },
     {
       "path": "packages/react-instantsearch/dist/umd/Connectors.min.js",
       "maxSize": "40.50 kB"
     },
     {
       "path": "packages/react-instantsearch/dist/umd/Dom.min.js",
-      "maxSize": "63.75 kB"
+      "maxSize": "63.00 kB"
     },
     {
       "path": "packages/react-instantsearch-core/dist/umd/ReactInstantSearchCore.min.js",
       "maxSize": "41.50 kB"
```

Comment:
there's also proptypes in this package no?

Reply:
Yes but those packages use the already transpiled version of the `react-instantsearch-core` & `react-instantsearch-dom` and I don't think Rollup runs Babel on them. I don't think it's an issue since this package is not documented anymore we only advocate for the platform specific package.